department-of-veterans-affairs/caseflow
# Issue with hearing_date not being properly set in UTC and with 00:00:00 time

Also need to make sure the time of 00:00:00 is preserved when delivering to client through JSON.

Connects #6472 

### Description
Hearing date should not have a time component to it, as it currently is in VACOLS. Also use UTC to manage the date value as VACOLS stores dates in UTC.

### Acceptance Criteria 
- [ ] Revised test cases pass

### Testing Plan
1. Make sure you have enough hearing schedule rows to test.
2. Select a start and end date in the View Schedule page
3. Review the results and make sure the list includes rows for the end date entered.
